dreem/inference/eval.py

```
@@ -26,24 +26,35 @@
-        checkpoint = eval_cfg.cfg.ckpt_path
+        hparams = {}
+
+    checkpoint = eval_cfg.cfg.ckpt_path
 
+    logger.info(f"Testing model saved at {checkpoint}")
     model = GTRRunner.load_from_checkpoint(checkpoint)
+
     model.tracker_cfg = eval_cfg.cfg.tracker
     model.tracker = Tracker(**model.tracker_cfg)
+
     logger.info(f"Using the following tracker:")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)